ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -4150,3 +4150,23 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_random_uniform_like) {
     test_case.add_input<ngraph::float16>(Shape{2, 2}, {41, 42, 43, 44});
     test_case.run();
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, onnx_model_bias_gelu) {
+    const auto function = onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/bias_gelu.onnx"));
+
+    auto test_case = test::TestCase<TestEngine>(function);
+    test_case.add_input<float>({0.5488135,
+                                0.71518934,
+                                0.60276335,
+                                0.5448832,
+                                0.4236548,
+                                0.6458941,
+                                0.4375872,
+                                0.891773,
+                                0.96366274,
+                                0.3834415});
+    test_case.add_input<float>({0.79172504, 0.5288949, 0.56804454, 0.92559665, 0.07103606});
+    test_case.add_expected_output<float>(
+        {1.2198428, 1.1112978, 1.0293297, 1.366493, 0.3411342, 1.329408, 0.8051748, 1.354462, 1.8336612, 0.3068893});
+    test_case.run(2);
```

Comment:
2 is default value of tolerance bits macro. Add some comment why it's fixed here.
[`void run(const size_t tolerance_bits = DEFAULT_FLOAT_TOLERANCE_BITS)`](https://github.com/openvinotoolkit/openvino/blob/11c42885be8fde221c587d63df61c378f23c0d23/ngraph/test/util/test_case.hpp#L165)

Reply:
good catch. I was just trying with different tolerance values that's why it's explicit here. I removed that now.